tensorflow/python/kernel_tests/nn_ops/depthwise_conv_op_base.py

```diff
@@ -992,6 +993,10 @@ def testDepthwiseConv2DInputGradCompare(self):
         continue
```

Comment:
The comment on 989 states ROCm doesn't support double, but now bf16 is also skipped. I don't know if bf16 is supported and whether this change allows TF to use it. If you don't know either, I'd add to the comment "It's unknown whether bf16 is supported".

And same for other places with this comment.